pages/api/signin.js

```diff
@@ -27,7 +30,7 @@ export default async function handleSignIn(nextRequest, nextResponse) {
           resolve();
         }, delayInMilliseconds);
       });
-      nextResponse.status(200).send({ ...response.data });
+      nextResponse.status(200).send({ ...response?.data });
```

Comment:
The optional chaining would still cause an error if response is not defined right? Since we cannot de-structure `undefined` object.

Reply:
Your'e right. Added a fix